### PR TITLE
build: refactors email capture

### PR DESCRIPTION
**Pull request by matthew-garrett**
- Updates EmailCapture component to be responsive and to simplify the code

### CODE REVIEW

File: packages/shared/web/src/email-capture-section/input-form.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+export const InputForm = ({ submitState, onInputChange, onSubmit, email }) => {
+  const theme = useTheme();
+
+  return (
+    <Box>
+      {submitState === 'pending' && (
+        <ProgressIndicator type="circular" size={32} />
+      )}
+      {submitState === 'success' && (
```
Comment (jeffywu) [inline packages/shared/web/src/email-capture-section/input-form.tsx:15]: Does this need to be capitalized?

Comment (matthew-garrett) [reply packages/shared/web/src/email-capture-section/input-form.tsx:15]: It doesn't have to be. I thought it worked to have it match the "YOUR EMAIL" LabelText that you see before you submit

Comment (matthew-garrett) [reply packages/shared/web/src/email-capture-section/input-form.tsx:15]: Lmk if you want me to change it

Comment (jeffywu) [reply packages/shared/web/src/email-capture-section/input-form.tsx:15]: All good, as long as it works

**Pull request merged**